packages/mobile/src/screens/track-screen/PremiumTrackCornerTag.tsx

```diff
@@ -0,0 +1,39 @@
+import type { PremiumConditions } from '@audius/common'
+
+import IconCollectibleCornerTag from 'app/assets/images/iconCollectibleCornerTag.svg'
+import IconLockedCornerTag from 'app/assets/images/iconLockedCornerTag.svg'
+import IconSpecialAccessCornerTag from 'app/assets/images/iconSpecialAccessCornerTag.svg'
+import { makeStyles } from 'app/styles'
+
+const useStyles = makeStyles(() => ({
+  icon: {
+    position: 'absolute'
+  }
+}))
+
+type PremiumTrackCornerTagProps = {
+  doesUserHaveAccess: boolean
+  isOwner: boolean
+  premiumConditions: PremiumConditions
+}
+
+export const PremiumTrackCornerTag = ({
+  doesUserHaveAccess,
+  isOwner,
+  premiumConditions
+}: PremiumTrackCornerTagProps) => {
+  const styles = useStyles()
+
+  if (isOwner) {
+    if (premiumConditions.nft_collection) {
+      return <IconCollectibleCornerTag style={styles.icon} />
+    }
+    return <IconSpecialAccessCornerTag style={styles.icon} />
+  }
+
+  if (doesUserHaveAccess) {
```

Comment:
Question for product - if the user has unlocked a track, do we not want to show them something to let them know that this is a special thing?

Reply:
we show them a label on the top right hand side of the tile, or in the big tile on track screen. they'll know it's a collectible gated or special access track